run.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Perhaps we should be more descriptive with the name of this script? Maybe something like runBackgroundProcesses.sh, or something along those lines?

Reply:
Renamed to runBEIMAProcesses.sh since it runs everything needed for BEIMA